homeassistant/components/group/config_flow.py

```diff
@@ -67,6 +77,32 @@ async def binary_sensor_options_schema(handler: SchemaCommonFlowHandler) -> vol.
     }
 )
 
+SENSOR_CONFIG_EXTENDS = {
+    vol.Required(CONF_TYPE): selector.SelectSelector(
+        selector.SelectSelectorConfig(options=_STATISTIC_MEASURES),
+    ),
+}
+SENSOR_OPTIONS = {
+    vol.Optional(CONF_IGNORE_NON_NUMERIC, default=True): selector.BooleanSelector(),
```

Comment:
IMHO, this should be `False` by default, to prevent unexpected results.

```suggestion
    vol.Optional(CONF_IGNORE_NON_NUMERIC, default=False): selector.BooleanSelector(),
```